pagopa/pn-frontend-e2e
# Pn 9245 e 2 e mittente invio notifica digitale a destinatario con domicilio di piattaforma nessun domicilio generale e nessuna pec fornita dalla pa caso ok al primo tentativo

## Short description

Created new test: PA send a notification and wait for status "consegnata"

## List of changes proposed in this pull request

- Created new test: PA send a notification and wait for the notification's state to be "Consegnata"
- This PR contains test 9244, too

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@TA_invioNotificaDestinatarioStatoConsegnata"` and verify if the test works fine.
